src/oneD/refine.cpp

```diff
@@ -12,7 +12,7 @@ namespace Cantera
 {
 Refiner::Refiner(Domain1D& domain) :
     m_ratio(10.0), m_slope(0.8), m_curve(0.8), m_prune(-0.001),
-    m_min_range(0.01), m_domain(&domain), m_npmax(3000),
+    m_min_range(0.01), m_domain(&domain), m_npmax(300),
```

Comment:
3000 may be somewhat excessive, but I think 300 is way too low, depending on the refining tolerances that you are using. I would be annoyed if I spent a long time waiting for a solution only to have it stop because it reached this arbitrary limit.